src/common/pico_time/include/pico/time.h

```diff
@@ -416,9 +416,11 @@ void alarm_pool_destroy(alarm_pool_t *pool);
  * @param time the timestamp when (after which) the callback should fire
  * @param callback the callback function
  * @param user_data user data to pass to the callback function
- * @param fire_if_past if true, this method will call the callback itself before returning 0 if the timestamp happens before or during this method call
- * @return >0 the alarm id
- * @return 0 the target timestamp was during or before this method call (whether the callback was called depends on fire_if_past)
+ * @param fire_if_past if true, this method will call the callback itself before returning if the timestamp happens before or during this method call
```

Comment:
Perhaps add a comma after "returning"? The first time I read it, I thought it meant that "if the timestamp happens before or during this method call" was what was being returned, IYSWIM.